js/templates/pageNav.html

```diff
@@ -15,30 +15,30 @@
         </div>
       </div>
       <div>
-        <div class="flexVCent">
+        <div class="flexVCent iconPad">
           <a class="iconBtn ion-chevron-left js-navBack"></a>
           <a class="iconBtn ion-chevron-right js-navFwd"></a>
           <a class="iconBtn ion-refresh js-navReload"></a>
         </div>
       </div>
       <div class="rowDivV clrBrBk"></div>
-      <div class="pageNavCenter">
+      <div class="pageNavCenter margL">
```

Comment:
Since the pageNavCenter class isn't used anywhere else, can't the margin just be part of that class?

To be consistent and avoid surprises, the right and left sides should both use padding or margin (right now the right side has padding as part of the pageNavCenter class).